e2e-tests/tests/features/dbbackup_pg/25-assert.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kuttl.dev/v1
+kind: TestAssert
+timeout: 400
+---
+apiVersion: everest.percona.com/v1alpha1
+kind: DatabaseCluster
+metadata:
+  name: test-pg-source
+status:
+  status: ready
+---
+apiVersion: pgv2.percona.com/v2
+kind: PerconaPGCluster
+metadata:
+  name: test-pg-source
+status:
+  state: ready
```

Comment:
should we check if the new secret is there as well?

Reply:
I'm not sure how we would check for this.
The new secret is going to be there - either created by us or the operator.
We would need to compare it's the same as the other one, but I don't think kuttl supports this type of check.

Reply:
Makes sense, ok